plc4j/integrations/opcua-server/src/main/java/org/apache/plc4x/java/opcuaserver/backend/Plc4xCommunication.java

```diff
@@ -115,9 +71,27 @@
     Map<NodeId, DataItem> monitoredList = new HashMap<>();
 
     public Plc4xCommunication () {
+
+    }
+
+    @Override
+    protected void onStartup() {
         driverManager = new PooledPlcDriverManager();
     }
 
+    @Override
```

Comment:
are you hiding the base class onShutdown?
If so, I would comment this.

Reply:
It overrides an abstract method, I don't really have much to do on shutdown. I've added a comment to indicate that.

Reply:
ah, IC,  you can't tell if it is abstract base or interface, always catches me.  thanks